sci-biology/open-ephys-gui/open-ephys-gui-0.5.5.ebuild

```
@@ -0,0 +1,79 @@
+# Copyright 2019-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake
+
+DESCRIPTION="Processing, recording, and visualizing multichannel ephys data"
+HOMEPAGE="http://www.open-ephys.org/gui/"
+LICENSE="GPL-3"
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/open-ephys/plugin-GUI"
+	EGIT_BRANCH="master"
+	Suffix=$EGIT_BRANCH
+	SubDir=${P}
+elif [[ ${PV} == "99999999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/open-ephys/plugin-GUI"
+	EGIT_BRANCH="development"
+	Suffix=$EGIT_BRANCH
+	SubDir=${P}
+else
+	SRC_URI="https://github.com/open-ephys/plugin-GUI/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	Suffix=${PV}
+	SubDir="plugin-GUI-${PV}"
+	S="${WORKDIR}/$SubDir"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+SLOT="${PV}"
+IUSE="jack"
+
+DEPEND="
+	media-libs/alsa-lib
+	media-libs/freeglut
+	media-libs/freetype
+	x11-libs/libXrandr
+	x11-libs/libXcursor
+	x11-libs/libXinerama
+	jack? ( || ( media-sound/jack-audio-connection-kit media-sound/jack2 ) )
+"
+RDEPEND="${DEPEND}"
+
+BUILD_DIR="$S/Build"
```

```suggestion
BUILD_DIR="${S}/Build"
```

Btw, why do we override this? Can't we use the default?

---

Unfortunately not. If we don't, I got this:
```bash
CMake Error at CMakeLists.txt:19 (message):
  Please run cmake inside the Build folder to build the binaries in the
  predefined paths.

  Set the OE_DONT_CHECK_BUILD_PATH cmake variable to TRUE to disable this
  error
```

So I guess it's easier to build it where it expects to be built :)